blog/16-03-23-create-deploy-and-mint-your-eRC223-contract-on-celo-with-hardhat/create-deploy-and-mint-your-eRC223-contract-on-celo-with-hardhat.md

```diff
@@ -0,0 +1,472 @@
+
+According to the ERC223 token standard contract creators, the basic motivation behind creating the contract was the massive amount of ERC20 tokens being lost on transactions when being transferred to an unintended or unsupported token contract. You can visit the [link](https://github.com/Dexaran/ERC223-token-standard) for more information about the contract specifications.
+
+
+![motivation](https://user-images.githubusercontent.com/69092079/225621959-df5ae0e9-cf45-4c44-adad-da04fb43bed6.png)
+
+
+## Using the ERC223 Token Standard
+
+1. Contract-Based Trading: ERC223 tokens can be used to facilitate contract-based trading. This allows users to create contracts with conditions that must be met for the trade to be completed. This helps ensure that both parties are satisfied with the terms of the trade. 
+
+2. Security Token Offerings: ERC223 tokens can be used to facilitate Security Token Offerings (STOs). These tokens represent a company’s assets and are backed by real-world assets. By using ERC223 tokens, companies can issue digital securities that are compliant with regulatory standards.
+
+3. Non-Fungible Tokens: Non-fungible tokens (NFTs) are unique tokens that can’t be interchanged. They represent digital assets such as art, music, and game items. ERC223 tokens can be used to facilitate the creation and trading of NFTs on the Ethereum blockchain. 	
+
+4. Decentralized Exchanges: ERC223 tokens can be used to facilitate decentralized exchanges. These exchanges allow users to buy and sell digital assets without the need for a third party. This
+5.  provides users with more control over their funds and can help reduce transaction costs. 
+
+5. Payment Systems: ERC223 tokens can be used to facilitate payment systems. This allows users to send and receive payments using digital assets on the Ethereum blockchain. This can be used to facilitate payments between individuals, companies, and organizations.
+
+6. Loyalty Token: The company can issue the ERC223 token to reward loyal customers by providing them with incentives such as discounts, cashback, or other rewards.
+
+7. Gaming Token: The gaming platform would use the ERC223 token as in-game currency, allowing players to earn rewards and trade items on the blockchain.
+
+8. Fundraising Token: ERC223 token can be used to raise funds for various projects, offering investors a stake in the project or access to exclusive benefits.
+
+9. Asset tokens: ERC223 token represent assets such as real estate or other physical assets, allowing for fractional ownership and easy transfer of ownership.
+
+10. Voting tokens: ERC223 tokens can be used for voting in decentralized autonomous organizations (DAO), giving token holders a say in the decision-making process.
+
+## Setting up Hardhat
```

Comment:
We are supposed to leave one line of space after each of the heading or subheading